PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -85,6 +86,11 @@ struct StoreAppConditionalView: View {
 
     @State private var cache = DataCache.instance
 
+    @Sendable private func waitForIconLoad() async {
+        try? await Task.sleep(nanoseconds: 6_000_000_000)
```

Comment:
`loadingLocalIcon` should be set to `true` at the start of the function, as the image might reload (such as during grid view) but would have the error icon instead of the loading one.